garrysmod/lua/includes/modules/markup.lua

```diff
@@ -251,17 +287,19 @@ function Parse( ml, maxwidth )
 	local thisMaxY = 0
 	local new_block_list = {}
 	local ymaxes = {}
+	local lineWidths = {}
 
 	local lineHeight = 0
-	for i, blk in pairs( blocks ) do
+	for i, blk in ipairs( blocks ) do
 
 		surface.SetFont( blocks[ i ].font )
 
+		blocks[ i ].text = string.gsub( blocks[ i ].text, "&gt;", ">" )

```

Comment:
This can be replace to a table as well, capture all `&tags;` if the tag is not found gsub just leaves it as it is

Reply:
Good point.